homework6/task1_counter/tests/test_instance_counter.py

```diff
@@ -0,0 +1,44 @@
+from project1_counter.instance_counter import instances_counter
+
+
+@instances_counter
+class User:
+    pass
+
+
+@instances_counter
+class Automobile:
+    instances_amount = "black"
+
+    def __init__(self, year):
+        self.year = year
+
+    def some_method(self):
+        return self.year
+
+
+def test_user_class_create_instances_and_reset_instance():
+    assert User.get_created_instances() == 0
+    user, _, _ = User(), User(), User()
+    assert user.get_created_instances() == 3
+    assert user.reset_instances_counter() == 3
+
+
+def test_user_create_instances_reset_and_then_check_instances_amount():
```

Comment:
It is better to describe not what are you doing (I can guess it from code) but hat are you want to check. 

```
test_reset_all_set_instance_to_zero
```